test_qtsass.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import unittest
+from textwrap import dedent
+from qtsass.qtsass import NotConformer, QLinearGradientConformer
```

Comment:
For imports lets break them on sections like

https://github.com/spyder-ide/loghub/blob/master/loghub/core/formatter.py#L12